server/src/main/java/org/opensearch/search/aggregations/bucket/BucketsAggregator.java

```diff
@@ -221,10 +221,6 @@ protected final InternalAggregations[] buildSubAggsForBuckets(long[] bucketOrdsT
         }
         InternalAggregations[] result = new InternalAggregations[bucketOrdsToCollect.length];
         for (int ord = 0; ord < bucketOrdsToCollect.length; ord++) {
-            InternalAggregation[] slice = new InternalAggregation[subAggregators.length];
-            for (int i = 0; i < subAggregators.length; i++) {
-                slice[i] = aggregations[i][ord];
-            }
```

Comment:
Was this intended to be used below? Is it slow to do `aggregations[index][ord]`?

Reply:
I checked the diff when this was added. I think it was just a mistake. 

I suspect that the previous logic was something like `result[ord] = slice`.